fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -495,7 +529,8 @@ ACTOR Future<Void> commitBatch(
 	}
 
 	/////// Phase 1: Pre-resolution processing (CPU bound except waiting for a version # which is separately pipelined and *should* be available by now (unless empty commit); ordered; currently atomic but could yield)
-	TEST(self->latestLocalCommitBatchResolving.get() < localBatchNumber-1); // Queuing pre-resolution commit processing 
+	TEST(self->latestLocalCommitBatchResolving.get() <
+	     localBatchNumber - 1); // Queuing pre-resolution commit processing
```

Comment:
I'm not sure whether our code coverage tools would allow a line split like this. I'm also not sure whether we use those tools anymore.

Reply:
Oh in that case I will just undo this format change.